src/docs/content/observability/observability-example/setup.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Setup"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+weight: 3
+tags:
+  - OCI Logging
+  - OCI Notifications
+  - OCI Service Connector Hub
+  - Deploy MuShop
+  - fluentd
+  - logging
+  - alarms
+  - notifications
+---
+
+### Introduction
+
+In this section we will deploy MuShop, Oracle Cloud Infrastructure (OCI) Notifications, Logging and Service Connector Hub.
+
+### Deploy MuShop
+
+{{% alert style="information" icon="warning" %}}
+We will be deploying MuShop on Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)
+{{% /alert %}}
+
+- Create kubernetes namespace
+```bash
+kubectl create namespace mushop
+```
```

Comment:
This is likely a step already performed by users, so it's important to designate as optional. And/or direct users to the cluster [setup](https://oracle-quickstart.github.io/oci-cloudnative/quickstart/kubernetes/#setup) steps